src/refresh-indicator.jsx

```diff
@@ -24,7 +24,7 @@ const RefreshIndicator = React.createClass({
 
     /**
      * Override the theme's color of the indicator while
-     * it's status is "loading" or it's percentage is 100.
+     * it's status is "loading".
```

Comment:
I changed the description to match the behaviour - so either the description was wrong (now corrected), the behaviour is wrong, or I made a mistake in my testing!